service/frontend/adminGrpcHandler.go

```diff
@@ -192,11 +192,11 @@ func (g adminGRPCHandler) UpdateGlobalIsolationGroups(ctx context.Context, reque
 }
 
 func (g adminGRPCHandler) GetDomainIsolationGroups(ctx context.Context, request *adminv1.GetDomainIsolationGroupsRequest) (*adminv1.GetDomainIsolationGroupsResponse, error) {
-	panic("not implemented")
-	return nil, nil
+	res, err := g.h.GetDomainIsolationGroups(ctx, proto.ToGetDomainIsolationGroupsRequest(request))
+	return proto.FromGetDomainIsolationGroupsResponse(res), proto.FromError(err)
 }
 
 func (g adminGRPCHandler) UpdateDomainIsolationGroups(ctx context.Context, request *adminv1.UpdateDomainIsolationGroupsRequest) (*adminv1.UpdateDomainIsolationGroupsResponse, error) {
-	panic("not implemented")
-	return nil, nil
+	res, err := g.h.UpdateDomainIsolationGroups(ctx, proto.ToUpdateDomainIsolationGroupsRequest(request))
+	return proto.FromUpdateDomainIsolationGroupsResponse(res), err
```

Comment:
nit: proto.FromError
And unrelated to this PR, `UpdateGlobalIsolationGroups` has the same problem

Reply:
👍